### PR TITLE
Bump version to 0.1.1 and add changelog

### PR DESCRIPTION
**Pull request by tahini**
Official release of 0.1.1

### CODE REVIEW

File: metadata.txt [hunk 1]
```diff
@@ -17,7 +17,16 @@ repository=https://github.com/chairemobilite/transition_qgis
 # Recommended items:
 
 hasProcessingProvider=no
-changelog=Version 0.1 (initial)
+changelog=
```
Comment (greenscientist) [inline metadata.txt:20]: We have a duplication of the changelog here. We should see if we could simply import one file.

**Review by greenscientist — APPROVED** [review]

File: metadata.txt (hunk 1 above)
Comment (tahini) [reply metadata.txt:20]: I looked at a few qgis plugins and they all have the changelog in the metadata so it is available in the qgis plugin registry. The main CHANGELOG is for github and honestly easier to edit. I don't know if we can put a link in the metadata's changelog. If so, we could keep the latest release and add a link to the complete changelog on github. But I haven't seen a plugin with a changelog link, but I haven't checked them all.

**Comment by greenscientist** [comment]
I would argue we could drop the Changelog.md and just point people to the metadata. 
(I personnally often find those changelog.md file useless on github, since it's often just a copy of the git history

**Comment by tahini** [comment]
I'l drop it in the next release then

**Pull request merged**